test/vaadin-button_test.html

```diff
@@ -100,6 +100,16 @@
 
         expect(vaadinButton.hasAttribute('active')).to.be.false;
       });
+
+      it('should not have active attribute when disabled', () => {
```

Comment:
`'should not have active attribute' => 'should not get an active attribute form keyboard events'`